code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -649,3 +650,19 @@
 		else
 			uniform = /obj/item/clothing/under/skirt/color/random
 
+/obj/item/storage/box/syndie_kit/chameleon/ghostcafe
+	name = "ghost cafe costuming kit"
+	desc = "Look just the way you did in life - or better!"
+
+/obj/item/storage/box/syndie_kit/chameleon/ghostcafe/PopulateContents() // Doesn't contain a PDA, for isolation reasons.
+	new /obj/item/clothing/under/chameleon(src)
+	new /obj/item/clothing/suit/chameleon(src)
+	new /obj/item/clothing/gloves/chameleon(src)
+	new /obj/item/clothing/shoes/chameleon(src)
+	new /obj/item/clothing/glasses/chameleon(src)
+	new /obj/item/clothing/head/chameleon(src)
+	new /obj/item/clothing/mask/chameleon(src)
+	new /obj/item/storage/backpack/chameleon(src)
+	new /obj/item/radio/headset/chameleon(src)
+	new /obj/item/stamp/chameleon(src)
```

Comment:
I think these two could and should be removed. Especially the headset.

Reply:
The stamp is harmless, and the lathe in the ghost bar can already make radio headsets (that don't work on the z-level anyway).

Reply:
Eeeeeeeh, just suppose we ay want to use this oufit for similar spawners at some point, that may work against it.